this patch installs the calligra fonts in the Debian way, in /usr/share/fonts/calligra
--- a/plugins/formulashape/fonts/CMakeLists.txt
+++ b/plugins/formulashape/fonts/CMakeLists.txt
@@ -1,5 +1,6 @@
 
-install( FILES  cmex10.ttf Arev.ttf ArevIt.ttf ArevBd.ttf ArevBI.ttf  DESTINATION ${DATA_INSTALL_DIR}/formulashape/fonts )
+# Debian: cmex10.ttf is already packaged by fonts-lyx
+install( FILES  Arev.ttf ArevIt.ttf ArevBd.ttf ArevBI.ttf  DESTINATION share/fonts/calligra/ )
 
 
 
--- a/plugins/musicshape/fonts/CMakeLists.txt
+++ b/plugins/musicshape/fonts/CMakeLists.txt
@@ -1 +1 @@
-install( FILES Emmentaler-14.ttf  DESTINATION ${DATA_INSTALL_DIR}/musicshape/fonts)
+install( FILES Emmentaler-14.ttf  DESTINATION share/fonts/calligra/)
